test/check-journal

```diff
@@ -149,18 +149,9 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
         })""")
 
         m.execute("systemctl start log123")
-
-        if "222-10" in m.execute("rpmquery systemd || true"):
-            # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1292805
-            # --follow is broken again, so we need to switch pages explicitly
-            # and hope that the service has started sending out messages in the meantime...
-            b.go("#/?prio=*&start=oldest&service=nonexisting.service")
-            wait_journal_empty()
-            b.go("#/?prio=*&start=oldest&service=log123.service")
+        b.go("#/?prio=*&start=oldest&service=log123.service")
```

Comment:
we should still be on this page
https://github.com/cockpit-project/cockpit/pull/3391/files#diff-573b224077eb8056f85d23afe25f9a6bR109
What we could do is move those two lines beneath the function definition.